lib/assignment/range.js

```diff
@@ -14,6 +14,9 @@ function assignRange (topicPartition, groupMembers, callback) {
 
   const topicMemberMap = topicToMemberMap(groupMembers);
   for (var topic in topicMemberMap) {
+    if (!topicMemberMap.hasOwnProperty(topic)) {
```

Comment:
I have applied the same fix everywhere, but it can probably be optimized in some cases